Sources/SwifterSwift/SwiftStdlib/DefaultStringInterpolationExtensions.swift

```diff
@@ -0,0 +1,46 @@
+// DefaultStringInterpolationExtensions.swift - Copyright 2023 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
```

Comment:
Why does it requires `Foundation`? `DefaultStringInterpolation` is defined on stdlib

Reply:
Updated, I also added the missing item for `DecodableExtensions.swift`.